app/pages/home-common/social.jsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import Translate from 'react-translate-component';
+import { Link } from 'react-router';
+import counterpart from 'counterpart';
+import moment from 'moment';
+import ProjectCard from '../../partials/project-card';
+import { getPublication, getRecentProjects, getBlogPosts, getNewestProject } from '../../lib/get-social-data';
+
+counterpart.registerTranslations('en', {
+  socialHomePage: {
+    community: 'The Zooniverse Community',
+    daily: 'The Daily Zooniverse',
+    news: 'News',
+    newestProject: 'Newest Project',
+    recentProjects: 'Recent Project Updates',
+    recentPublications: 'Recent Publications'
+  }
+});
+
+export default class HomePageSocial extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      blogPosts: [],
+      newestProject: {},
+      newestPublication: {},
+      recentProjects: []
+    };
+  }
+
+  componentDidMount() {
+    this.getSocial();
+  }
+
+  getSocial() {
+    getNewestProject().then((newestProject) => {
+      this.setState({
+        newestProject,
+        newestPublication: getPublication()
+      });
+    });
+    getRecentProjects().then((recentProjects) => {
+      this.setState({ recentProjects });
+    });
+    getBlogPosts((blogPosts) => {
+      this.setState({ blogPosts });
+    });
+  }
+
+  renderUpdatedProject(project) {
+    const link = `/projects/${project.slug}`;
+    const timestamp = moment(new Date(project.updated_at)).fromNow();
+    return (
+      <div key={project.id}>
+        <h4 className="timestamp-label">{timestamp}</h4>
+        <h5 className="tertiary-headline">{project.display_name} </h5>
+        <a className="home-social__italic-link" href={link}> View Project </a>
+        <hr />
+      </div>
+    );
+  }
+
+  renderBlogPost(post, i) {
+    const background = {};
+    background.backgroundImage = `url(${post.image})`;
+    if (!post.image) { background.display = 'none'; }
+    const firstPost = i === 0 ? 'home-social__blog-section--white' : 'home-social__blog-section--gray';
```

Comment:
You can require in [`classnames`](https://www.npmjs.com/package/classnames) for conditional CSS classes. It might be slightly cleaner than string interpolation.